pkgs/development/libraries/oniguruma/default.nix

```diff
@@ -1,24 +1,27 @@
-{ lib, stdenv, fetchFromGitHub, autoreconfHook }:
+{ lib, stdenv, fetchurl, autoreconfHook }:
+
+# Note: do not use fetchpatch or fetchFromGitHub to keep this package available in __bootPackages
```

Comment:
Maybe put this near `src`, and add `passthru.tests` so ofborg will catch a problem if it arises?

Reply:
I would rather lift the tests one lever higher than limiting fetchpatch and so on in more packages.

Reply:
The developer experience of stdenv is worth more than lifting fetcher function restrictions in only two packages.